index.d.ts

```diff
@@ -139,7 +144,7 @@ declare namespace ansiStyles {
 
 		@param hex - A hexadecimal string containing RGB data.
 		*/
-		hexToRgb(hex: string): [red: number, green: number, blue: number];
+		hexToRgb(hex: string): [number, number, number];
```

Comment:
Don't do unrelated changes

Reply:
The tests seemed to fail without this change, but I will revert it.